wagtail/tests/test_page_queryset.py

```diff
@@ -919,6 +919,78 @@ def test_specific_query_with_annotation(self):
         self.assertEqual(results.first().subscribers_count, 1)
         self.assertEqual(results.last().subscribers_count, 1)
 
+    def test_specific_subquery_select_related(self):
+        pages = list(
+            Page.objects.type(EventPage)
+            .specific()
+            .select_related("feed_image", for_specific_subqueries=True)
```

Comment:
Suggestion: I'd be interested in a test to see how calling `.select_related(for_specific_subqueries=False)` interacts with having specific values set. It'd be good to confirm which queryset gets which additions.

Reply:
That's not something I want to support - which is why `select_related()` is currently only applied to subqueries when actual relationships have been specified. Supporting blanket `select_related()` is possible, but it gets complicated when it comes to respecting chained `select_related()` applications.

Reply:
One for a future PR, I think!